test/clj/rems/api/test_permissions.clj

```diff
@@ -13,7 +13,7 @@
             [schema.core :as s]))
 
 (use-fixtures
-  :once
+  :each
```

Comment:
Did this fix some stability problem? I think the reason why the API wasn't being constructed in each was that it's just somewhat slower.

Reply:
`api-fixture` (eventually) resets e.g. user and role caches, which need to be cleaned up before each deftest.